src/python/director/drakevisualizer.py

```diff
@@ -200,6 +200,44 @@ def __init__(self, name, geom, polyData, parentTransform):
             self.polyDataItem.shadowOn()
 
 
+class LidarSource(TimerCallback):
```

Comment:
I know there are examples in the code base where we inherit from TimerCallback, but I prefer not to do that anymore.  Instead it can be something like:

    self.timer = TimerCallback(targetFps=30)
    self.timer.callback = self._updateSource